java-frontend/src/test/java/org/sonar/java/SonarComponentsTest.java

```diff
@@ -451,4 +453,16 @@ public void feedback_should_not_be_sent_in_sonarLintContext_or_when_collecting_i
     assertThat(sensorContext.measure("projectKey", "sonarjava_feedback")).isNull();
 
   }
+
+  @Test
+  public void test_generation_of_ucfg() {
```

Comment:
@benzonico Could we rename test method to something like `ucfg_activation_should_rely_on_active_rules`? (now name is misleading a bit)